index.html

```diff
@@ -3525,19 +3525,19 @@ <h5>JWT Encoding</h5>
 
             <p>
 If no explicit rule is specified, <a>properties</a> are encoded in the same way
-as with a standard <a>verifiable credential</a>, and are added to the
```

Comment:
This change should be accepted for the reasons I gave in https://github.com/w3c/vc-data-model/pull/828/files#r753698782